README.md

```diff
@@ -50,13 +51,13 @@ A Discord.js based Discord chat bot. 🤖
 
 ## <a id="running-debugging-the-project"></a> Running/Debugging the project
```

Comment:
🚓 👮  Markdown rule violation 🚔  `MD033/no-inline-html`

Check warnings through `yzhang.markdown-all-in-one` extension and auto-update *Table of Contents*